eng/dependabot/Packages.props

```diff
@@ -61,7 +61,7 @@
     <PackageVersion Include="Verify.Xunit" Version="19.14.1" />
     <PackageVersion Update="Verify.XUnit" Condition="'$(VerifyXUnitVersion)' != ''" Version="$(VerifyXUnitVersion)" />
 	
-	<PackageVersion Include="Microsoft.VisualStudio.SolutionPersistence" Version="$(MicrosoftVisualStudioSolutionPersistenceVersion)" />
+	  <PackageVersion Include="Microsoft.VisualStudio.SolutionPersistence" Version="$(MicrosoftVisualStudioSolutionPersistenceVersion)" />
```

Comment:
Formatting change: 👍🏻 

But should this follow the pattern of the rest of everything in this file? It doesn't matter much while Dependabot is broken, I guess, but I'd rather be able to move forward. Though I think maybe with source-build requirements we can't really do the dependabot thing anyway?

Reply:
if dependabot won't update this package, then i think we should move this. where do you think would be best place?

Reply:
Dependabot is just straight-up not working for us at all:  https://github.com/dependabot/dependabot-core/issues/7206. Nothing unique to this package.

Reply:
maybe in `Microsoft.Build.csproj` right next to
https://github.com/dotnet/msbuild/blob/b1ae836166204e2f4d44f46af4b6b3d4ebec6660/src/Build/Microsoft.Build.csproj#L33
?

Reply:
okay, then leaving it as it is.